pkg/apis/migration/v1alpha1/model.go

```diff
@@ -131,6 +132,38 @@ func GetStorage(client k8sclient.Client, ref *kapi.ObjectReference) (*MigStorage
 	return &object, err
 }
 
+// Get a referenced Migration for DVM.
+// Return nil if the reference cannot be resolved.
+func GetMigrationForDVM(client k8sclient.Client, owners []v1.OwnerReference) (*MigMigration, error) {
+	if len(owners) == 0 {
+		return nil, nil
+	}
+	migrationName := owners[0].Name
+	migrationObject := MigMigration{}
+	err := client.Get(context.TODO(),
+		k8sclient.ObjectKey{
+			Name:      migrationName,
+			Namespace: OpenshiftMigrationNamespace,
+		}, &migrationObject)
+	//objectList := MigMigrationList{}
+	//err := client.List(context.TODO(), &k8sclient.ListOptions{
+	//	LabelSelector: k8sLabels.SelectorFromSet(dvmLabels),
+	//}, &objectList)
+
+	if err != nil {
+		if errors.IsNotFound(err) {
+			return nil, nil
+		} else {
+			return nil, err
```

Comment:
Let's use `liberr.wrap(err)` so that the full stack trace will be returned down the call chain when this error is surfaced later.